script.js

```diff
@@ -1,110 +1,175 @@
-const template = document.getElementsByTagName("template")[0];
-const rootElem = document.getElementById("root");
-const bodyElem = document.getElementsByTagName("body")[0];
-
-let allEpisodes = []
-const api = 'https://api.tvmaze.com/shows/82'
-
-fetchMessage("Loading episodes")
-
-// fetch data from tvmaze and setup the page
-fetch(`${api}/episodes`)
-  .then((response) => response.json())
-  .then((data) => {
-    if (data.length > 100) {
-      for (let i = 0; i < 100; i++) {
-        allEpisodes.push(data[i])
-      }
-    } else {
-      allEpisodes = [...data];
-    }
-
-    fetchMessage("");
-  })
-  .then(() => makePageForEpisodes(allEpisodes))
-  .then(() => setupSearch())
-  .catch((error) => fetchMessage(error));
-
-// Set the correct series name
-fetch(api)
-  .then((response) => response.json())
-  .then((data) => setSeriesName(data.name))
-
-function setSeriesName(seriesName) {
-  const seriesNameField = document.querySelector("#series-name");
-  seriesNameField.innerText = seriesName;
+document.addEventListener('DOMContentLoaded', function () {
+  const defaultShowId = 82; // We're setting a number to identify a TV show. It's like giving the TV show a special code.
```

Comment:
I believe the best approach is to display a message asking the user to select a show, rather than load a default show.